### ISSUE TITLE
Missing typings for `Context`

### ISSUE THREAD

**Issue by tobira-shoe** [telegraf/telegraf#459]
## Context

* Telegraf.js Version: 3.12.2
* Node.js Version: 9.9.0
* Operating System: Windows

## Expected Behavior

Type definitions for documented [here](https://telegraf.js.org/#/?id=context) `Context` properties.

## Current Behavior

There some missing definitions for:
- `webhookReply`
- `updateType`
- `updateSubTypes`
- `match`

## To slove
To existed `Context` [interface](https://github.com/telegraf/telegraf/blob/develop/typings/index.d.ts#L22) add those missing definitions

```ts
{
  updateType: tt.UpdateType
  updateSubTypes?: tt.MessageSubTypes
  match?: RegExpMatchArray
}
```
Note: i did not found the way to add definition for `webhookReply`.

**Comment by FedeBert** [comment]
Did you manage to import the Context module?
How did you do?

I was thinkign about making a typescript wrapper for the class, but if there's a simpler and  easier way i'm into it!

**Comment by EdJoPaTo** [comment]
`ctx.match` is already supported since #684
`ctx.updateType` is supported since #493
`ctx.updateSubTypes` is included with #733 

I have not used `ctx.webhookReply`. Maybe someone else can add that.
